### ISSUE TITLE
Error after updating to Typescript 2.7

### ISSUE THREAD

**Issue by dderevjanik** [gillchristian/io-ts-reporters#10]
Hi, after updating to Typescript 2.7 (from 2.6.1) i'm getting this error:

```
ERROR in /home/PATH_TO_PROJECT/node_modules/io-ts-reporters/node_modules/io-ts/lib/index.d.ts
(213,31): error TS2344: Type '[P[K], Type<undefined>]' does not satisfy the constraint '[Type<any>]'.
  Types of property 'length' are incompatible.
    Type '2' is not assignable to type '1'.
```

**Comment by OliverJAsh** [comment]
Hi @dderevjanik,

The issue appears to be originating from io-ts.

This will most likely be fixed by upgrading io-ts https://github.com/OliverJAsh/io-ts-reporters/pull/9.

**Comment by OliverJAsh** [comment]
This should be fixed in v0.0.19 by https://github.com/OliverJAsh/io-ts-reporters/commit/6ec0051ae2786c263ad638e3a9e3d75fd8f7e0c0
